control/controlclient/direct.go

```diff
@@ -1414,20 +1425,20 @@ func postPingResult(now time.Time, logf logger.Logf, c *http.Client, pr *tailcfg
 		return err
 	}
 	// Send the results of the Ping, back to control URL.
```

Comment:
PingResult doesn't distinguish between TSMP and disco it seems? More reason to make a new JSON POST body type in tailcfg.